plugins/json-logger/README.md

```diff
@@ -0,0 +1,41 @@
+# @amplication/plugin-<name>json-logger
```

Comment:
Let's update the name properly. Plus can you rename the folder to `logger-json` instead? This will follow the convention set by the other plugins.

```suggestion
# @amplication/plugin-logger-json
```